libp2p/NodeTable.h

```diff
@@ -207,7 +207,8 @@ class NodeTable : UDPSocketEvents
         std::list<std::weak_ptr<NodeEntry>> nodes;
     };
 
-    /// Used ping known node. Used by node table when refreshing buckets and as part of eviction process (see evict).
+    /// Used ping known node. Used by node table when refreshing buckets and as part of eviction
```

Comment:
Not sure I understand `Used ping known node.` is it a typo?

Reply:
Yea me neither, I think it’s a typo from a previous commit. Will change.